### PR TITLE
XIVY-6828 Add product related data

### PR DESCRIPTION
**Pull request by alexsuter**
I like it all. But I'm just not sure about the folder name:

**product** vs. **excel-connector-product**

The **artifactId** is called **excel-connector-product**

I would prefer excel-connector-product. But will make templating a bit more harder ...

Resulting zip:
grafik

Repo layout:
grafik

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -10,6 +10,7 @@
     <module>${project.name}</module>
     <module>${project.name}-demo</module>
     <module>${project.name}-test</module>
+    <module>product</module>
```
Comment (ivy-rew) [inline pom.xml:13]: ${project.name}-product  looks much better IMHO ... I'd always prefer a meaningful name in the directories

Comment (ivy-rew) [reply pom.xml:13]: still we could opt for a 'product' directory in the template ... doesn't hurt us I think

File: README.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# Excel Connector
+
+[![CI Build](https://github.com/axonivy-market/excel-connector/actions/workflows/ci.yml/badge.svg)](https://github.com/axonivy-market/excel-connector/actions/workflows/ci.yml)
```
Comment (ivy-rew) [inline README.md:3]: cool to have technical information here at a glance :1st_place_medal:

File: product/zip.xml [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+    <includeBaseDirectory>false</includeBaseDirectory>
+
+    <formats>
+        <format>zip</format>
+    </formats>
+
+    <fileSets>
+        <fileSet>
+            <directory>.</directory>
+            <excludes>
```
Comment (ivy-rew) [inline product/zip.xml:12]: I think a whitelist would serve use better. It implicitly documents the expected content:

<include>product.json</include>
<include>README.md</include>
<include>**/*.png</include>

**Review by ivy-rew — APPROVED** [review]
feels damn good

File: product/zip.xml (hunk 3 above)
Comment (alexsuter) [reply product/zip.xml]: I think this is good!

File: pom.xml (hunk 1 above)
Comment (alexsuter) [reply pom.xml]: renamed to excel-connector-product

**Comment by alexsuter** [comment]
grafik

**Pull request merged**